### ISSUE TITLE
Can openvino load the model only once during inference (cpu) in C++ ? 

### ISSUE THREAD

**Issue by ChesterXcw** [openvinotoolkit/openvino#12036]
When deploy yolov5n , it takes about 250ms to initialize the model once (only call  from "ReadNetwork" to "LoadNetwork" take about 250ms, especially "LoadNetwork" costs  about 197ms ) , but not found any documents loading the model to memory (cpu)  or reducing init time   , only some blogs talk about GPU loading .
and my code are following:

bool Detector::init(string xml_path, double cof_threshold,double nms_area_threshold){

    _xml_path = xml_path;
    _cof_threshold = cof_threshold;
    _nms_area_threshold = nms_area_threshold;
    Core ie;
    CNNNetwork cnnNetwork = ie.ReadNetwork(xml_path); 

    //输入设置 
    InputsDataMap inputInfo(cnnNetwork.getInputsInfo());
    InputInfo::Ptr& input = inputInfo.begin()->second;
    _input_name = inputInfo.begin()->first;
    input->setPrecision(Precision::FP32);
    input->getInputData()->setLayout(Layout::NCHW);
    ICNNNetwork::InputShapes inputShapes = cnnNetwork.getInputShapes();
    SizeVector& inSizeVector = inputShapes.begin()->second;
    cnnNetwork.reshape(inputShapes);

    //输出设置
    _outputinfo = OutputsDataMap(cnnNetwork.getOutputsInfo());
    for (auto &output : _outputinfo) {
        output.second->setPrecision(Precision::FP32);
    }
    //获取可执行网络
    //_network =  ie.LoadNetwork(cnnNetwork, "GPU");
    _network =  ie.LoadNetwork(cnnNetwork, "CPU");

    return true;

int main(int argc, char const *argv[]){

    auto start_main = chrono::high_resolution_clock::now();
    Detector* detector = new Detector;
    string xml_path = "/best_openvino_model/best.xml";

    detector->init(xml_path,0.25,0.45);

    string img_path = argv[1];
    Mat src = imread(img_path);
    Mat osrc = src.clone();
    resize(osrc,osrc,Size(640,640));
     
    vector<Detector::Object> detected_objects;
    auto start = chrono::high_resolution_clock::now();
    
    detector->process_frame(src,detected_objects);
    
    auto end = chrono::high_resolution_clock::now();
    auto diff = std::chrono::duration_cast<std::chrono::milliseconds>(end - start);
    std::cout<<"inference cost "<<diff.count() <<" ms" << std::endl;
    return 0;

code reference  https://github.com/fb029ed/yolov5_cpp_openvino/blob/e95a9dcf814863c6dbe242ef9770f322aa64ad66/demo/detector.cpp#L64. 

openvino version: 2021.4.2 LTS

can someone tell me how to do ? thanks a lot .

**Comment by brmarkus** [comment]
Can you rephrase your question maybe?
Your code-snipped shows only one call to "LoadNetwork()", but you ask to load it only once?
Do you mean both calls, "ReadNetwork()" and "LoadNetwork()" take about 250ms?
Is it only for doing inference on CPU?

You might want to have a look into "ExecutableNetwork::export()" (https://docs.openvino.ai/2021.4/api/ie_python_api/_autosummary/openvino.inference_engine.ExecutableNetwork.html#openvino.inference_engine.ExecutableNetwork.export) and "IECore::import_network()" (https://docs.openvino.ai/2021.4/api/ie_python_api/_autosummary/openvino.inference_engine.IECore.html#openvino.inference_engine.IECore.import_network).

For loading model&weights from memory into OpenVINO, hvae a look at "https://docs.openvino.ai/2021.4/openvino_docs_IE_DG_protecting_model_guide.html".

**Comment by ChesterXcw** [comment]
> Can you rephrase your question maybe? Your code-snipped shows only one call to "LoadNetwork()", but you ask to load it only once? Do you mean both calls, "ReadNetwork()" and "LoadNetwork()" take about 250ms? Is it only for doing inference on CPU?
> 
> You might want to have a look into "ExecutableNetwork::export()" (https://docs.openvino.ai/2021.4/api/ie_python_api/_autosummary/openvino.inference_engine.ExecutableNetwork.html#openvino.inference_engine.ExecutableNetwork.export) and "IECore::import_network()" (https://docs.openvino.ai/2021.4/api/ie_python_api/_autosummary/openvino.inference_engine.IECore.html#openvino.inference_engine.IECore.import_network).
> 
> For loading model&weights from memory into OpenVINO, hvae a look at "https://docs.openvino.ai/2021.4/openvino_docs_IE_DG_protecting_model_guide.html".

sorry , i did not make it clear, you are right , calls, "ReadNetwork()" and "LoadNetwork()" take about 250ms, and i debug it ,only calls "LoadNetwork" takes about 197ms. at present, it only inferences on cpu.

**Comment by ChesterXcw** [comment]
> You might want to have a look into "ExecutableNetwork::export()" (https://docs.openvino.ai/2021.4/api/ie_python_api/_autosummary/openvino.inference_engine.ExecutableNetwork.html#openvino.inference_engine.ExecutableNetwork.export) and "IECore::import_network()" (https://docs.openvino.ai/2021.4/api/ie_python_api/_autosummary/openvino.inference_engine.IECore.html#openvino.inference_engine.IECore.import_network).
> 
> For loading model&weights from memory into OpenVINO, hvae a look at "https://docs.openvino.ai/2021.4/openvino_docs_IE_DG_protecting_model_guide.html".

Thanks, I will look carefully

**Comment by ChesterXcw** [comment]
Maybe find the solutions
>https://docs.openvino.ai/2021.4/openvino_docs_IE_DG_Model_caching_overview.html?sw_type=switcher-cpp
